src/Miew.js

```diff
@@ -2934,6 +2922,59 @@ Miew.prototype.setPivotAtom = function (atom) {
   this.dispatchEvent({ type: 'transform' });
 };
 
+Miew.prototype.setPivotSubset = (function () {
+  const _center = new THREE.Vector3(0.0, 0.0, 0.0);
+
+  function _includesInCurSelection(atom, selectionBit) {
+    return atom._mask & (1 << selectionBit);
+  }
+
+  function _includesInSelector(atom, selector) {
+    return selector.selector.includesAtom(atom);
+  }
+
+  return function (selector) {
+    const pos = this._gfx.pivot.position;
```

Comment:
Hey! I have a few comments about the function.
At first, I suggest dividing it into two logical parts:
-  search selected center point in visual
-  search center point for all visuals
These operations are similar. Maybe we can create a general function for the search? Otherwise using similar names for counts and positions confuses me as a reader.

Reply:
Thank for your comment
Divide this function on two different: one is the method of complex Visual class and another - of view
Cooperate them into one function produces complex auxiliary code